sdk/lusid/utilities/api_client_builder.py

```diff
@@ -52,17 +52,27 @@ def build(cls, api_secrets_filename=None, id_provider_response_handler=None, api
         # Load the configuration
         configuration = ApiConfigurationLoader.load(api_secrets_filename)
 
+        # If user passes a secrets file, then ignore the PAT loaded from env var
```

Comment:
i don't think you need this check, can't you just set the local `api_token` based on whether one is based in or not?